services/gateway-api-crds/1.6.0/gateway-api-crds.yaml

```diff
@@ -8,11 +8,11 @@ spec:
   chart:
     spec:
       chart: traefik-crds
-      version: "1.2.0"  # Use the appropriate version for Traefik CRDs
+      version: "1.6.0" # Keep this consistent with /services/traefik/<version>/crds/crds.yaml HelmRelease.
```

Comment:
maybe there's something we can do with the chart bumper script to bump this here whenever traefik is bumped